client/components/site-verticals-suggestion-search/README.md

```diff
@@ -58,4 +52,9 @@ Returns _{Object}_:
 
 ```
 
+### _(Boolean)_ `showPopular` 
+_Optional_ Informs the component whether to show a list of popular vertical topics when the input field is empty. Default: `false`
+
+### _(Boolean)_ `autoFocus` 
+_Optional_ When set to `true` gives immediate focus to the search input field. Default: `false`
```

Comment:
Would be great if we maintain a thorough list of all props here. Definitely don't consider that as a requirement though - it's something that can be followed up with in another PR.

Reply:
I've taken the approach that the README is for the consumer of the component, and therefore lists the props required to embed it, not necessarily those defined via connect, which are only important for internals. 

Having said that, happy to add them later. I'm sure I'll be back in this neck of woods very soon! :)